scss/config/_colors.scss

```diff
@@ -62,17 +62,23 @@ $neutral-colors: (
   'light-gray': '#brand-blue'
     (
       'tint': 80%,
-      'desaturate': 80%,
+      'color.scale': (
```

Comment:
Should these be using `color.adjust` instead?

Reply:
@jgerigmeyer in our case since we want to match the colors we have already established and tested, `color.adjust` will give us the same colors we had before.

Reply:
Fixed in https://github.com/oddbird/sassdoc-theme-herman/pull/449/commits/564f6638e44068715119475d462b944826336bcf